sys/fs/cd9660/cd9660_vnops.c

```diff
@@ -169,9 +192,26 @@ cd9660_getattr(struct vop_getattr_args *ap)
 	vap->va_fileid	= ip->i_number;
 
 	vap->va_mode	= ip->inode.iso_mode;
+	switch (vp->v_type) {
+	case VDIR:
+		if (ip->i_mnt->im_flags & ISOFSMNT_DMASK)
```

Comment:
Hmm, these also assign the mask instead of anding the mask.  In the case of an image with rock-ridge attributes this can result in increased permissions which doesn't match the description in the manpage changes.

Reply:
Because this is about enforcing the fmask.  Will check again the whole thing in the manpage changes.

Reply:
Fixed.